pom.xml

```diff
@@ -33,10 +33,12 @@
     </licenses>
 
     <properties>
-        <jenkins.version>2.7.3</jenkins.version>
-        <java.level>7</java.level>
-        <jackson.version>2.4.0</jackson.version>
-        <workflow-api.version>2.22</workflow-api.version>
+        <revision>1.10</revision>
+        <changelist>-SNAPSHOT</changelist>
+        <jenkins.version>2.138.4</jenkins.version>
+        <java.level>8</java.level>
+        <workflow-api.version>2.34-rc889.401891b554d9</workflow-api.version> <!-- TODO https://github.com/jenkinsci/workflow-api-plugin/pull/89 -->
+        <workflow-support.version>3.3-SNAPSHOT</workflow-support.version>
```

Comment:
Incremental version?

Reply:
Ah whoops, the update was just from debugging an unrelated issue. Will revert to 3.2.

Reply:
Done in bfe12f9.